### PR TITLE
Creating a bucket is necessary before adding it to remote storage

### LINKED ISSUE #415: missing information about s3 usage
Docs don't explicitly say that when one is trying to use s3 or s3 compatible storage there should already exist bucket.

Then, if you would try to push data to s3 and bucket wouldn't exist, you would get the following:
```
Preparing to upload data to 's3://my/remote/'
Preparing to collect status from s3://my/remote/
[############                  ] 40% Collecting information
ERROR: failed to push data to the cloud - An error occurred (NoSuchBucket) when calling the ListObjectsV2 operation: Unknown
``` 

So I propose to explicitly say in docs that you should already have existing bucket to be able to push to s3.
Following docs are related to s3 usage:
https://dvc.org/doc/get-started/configure
https://dvc.org/doc/commands-reference/remote-add
Related discussion in ods:
https://opendatascience.slack.com/archives/CGGLZJ119/p1559831715009900

### PR DESCRIPTION
**Pull request by Naba7**
Fix #415 
I just added the information to create a bucket with the links to it in commands-reference/config and commands-reference/remote .

### CODE REVIEW

File: static/docs/commands-reference/config.md [hunk 1]
```diff
@@ -181,6 +181,10 @@ $ dvc config core.loglevel debug
 
 Add an S3 remote and set it as the project default:
 
+Before,adding a new remote be sure to login into AWS services and follow
```
Comment (shcheklein) [inline static/docs/commands-reference/config.md]: use `>` before each line to make it a "note"

**Comment by Naba7** [comment]
> have you run the pre-commit hook? or prettier manually? I think links should be on a separate line.

No,I hadn't run pre-commit hook. Yes, I ran it just now, and I am changing the link to a separate line.

File: static/docs/commands-reference/config.md (hunk 1 above)
Comment (Naba7) [reply static/docs/commands-reference/config.md]: ok

File: static/docs/commands-reference/config.md [hunk 2]
```diff
@@ -181,6 +181,11 @@ $ dvc config core.loglevel debug
 
 Add an S3 remote and set it as the project default:
 
+> **Note!** Before,adding a new remote be sure to login into AWS services and
```
Comment (shcheklein) [inline static/docs/commands-reference/config.md]: Before,adding -> Before adding

Comment (Naba7) [reply static/docs/commands-reference/config.md]: fixed it

**Comment by shcheklein** [comment]
@Naba7 thanks 🎉 !

**Pull request merged**